### DISCUSSION TITLE
what is the goal of _beginthreadex  in ...\emscripten\.....\thread_win32.cpp

### DISCUSSION THREAD

**Discussion by aharondavid** [emscripten-core/emscripten#16764] [Q&A]
Hi,
wondering, what is the goal of _beginthreadex  in the file:
C:\Users\Administrator\emsdk\upstream\emscripten\system\lib\libcxx\src\support\win32\thread_win32.cpp
actually, what is the goal of all **win32** folder?

**Comment by sbc100** [comment]
This code is not used in emscripten.. its part of the libcxx code which comes from llvm but we don't ever compile it on our platform.

**Reply by aharondavid** [reply]
Ok, 
Thanks!
